tasks/main.yml

```diff
@@ -102,7 +102,7 @@
 
 - name: Clone OpenDroneMap repository {{ opendronemap_work_dir }}
   git:
-    repo=https://github.com/OpenDroneMap/OpenDroneMap
+    repo=https://github.com/qedsoftware/OpenDroneMap
     dest="{{ opendronemap_work_dir }}"
     version=v0.1
```

Comment:
The version `v0.1` will still be used, so effectively this pull request does change nothing.